task_sdk/src/airflow/sdk/api/datamodels/_generated.py

```diff
@@ -116,6 +116,18 @@ class ValidationError(BaseModel):
     type: Annotated[str, Field(title="Error Type")]
 
 
+class VariablePostBody(BaseModel):
+    """
+    Request body schema for creating variables.
+    """
+
+    model_config = ConfigDict(
+        extra="forbid",
+    )
```

Comment:
Nit
```suggestion
    model_config = ConfigDict(extra="forbid")
```


(The trailing comma makes Ruff split it over multiple lines.)

Reply:
Oh, this is generated. Nevermind.